ross/fluid_flow/fluid_flow_graphics.py

```diff
@@ -2,7 +2,6 @@
 import numpy as np
 from bokeh.plotting import figure
 from matplotlib import cm
-from mpl_toolkits.mplot3d import axes3d
```

Comment:
Although it seems this import is not used, we cannot remove this line, once we plotting 3D with matplotlib, we need the axes3d. In `results.py`, it happens the same.

Reply:
Here, we could do something like:
```
# fmt: off
from mpl_toolkits.mplot3d import axes3d
# fmt: on
```

Reply:
This import was removed with Pycharm's Optimize imports.
I think this can be solved by just adding the import!

Reply:
> This import was removed with Pycharm's Optimize imports.
I think this can be solved by just adding the import!

It's good to go now!